### DISCUSSION TITLE
Octernships Opportunities

### DISCUSSION THREAD

**Discussion by ayushsisodia7** [community/community#58624] [GitHub Education]
### Hi everyone,

I am currently not able to enroll in any opportunity, so I wanted to know if different Octernship Opportunities are available throughout the year?

**Comment by AlbinXXX** [comment]
Hi there!

Regarding your query, Octernship Opportunities may vary throughout the year. While I don't have specific information about the availability of Octernships at this moment, it's worth noting that Octernship programs typically have their own schedules and durations. It's advisable to regularly check the relevant platforms or websites that offer Octernship Opportunities to stay updated on their availability.

Here are a few platform suggestions where you can explore internship and Octernship opportunities:

LinkedIn: LinkedIn is a popular professional networking platform where you can find internship and Octernship listings. Many companies and organizations post their openings on LinkedIn, making it a valuable resource for job seekers.

Indeed: Indeed is a widely used job search platform that aggregates internship and Octernship opportunities from various sources. It allows you to filter and search for positions based on your preferences and location.

Glassdoor: Glassdoor not only provides insights into company reviews and salaries but also offers internship and entry-level job listings. It's a helpful platform to explore Octernship opportunities and learn more about potential employers.

In addition to these platforms, I also recommend joining the 100Devs Discord community (https://discord.gg/100devs) It's a great resource for finding internship and Octernship opportunities, as they actively post about such openings in their community.

Best of luck in your search for Octernship Opportunities, and I hope you find the perfect opportunity to further your learning and growth!

**Reply by ayushsisodia7** [reply]
Thank you for the information, will definitely take it under consideration
